graylog2-server/src/main/java/org/graylog2/plugin/lookup/LookupResult.java

```diff
@@ -56,11 +68,13 @@
     @Nullable
     public abstract Map<Object, Object> multiValue();
 
+
+    @JsonProperty("has_error")
+    public abstract boolean hasError();
```

Comment:
I was wondering if we might want to use an error message (or a list of error messages) instead of a simple flag. That way consumers of the lookup table API would be able to better communicate issues to the user. (e.g. showing the error message in the frontend for debugging)

What do you think?

Reply:
Keeping it a boolean has the advantage that we can always return the same Object (`ERROR_LOOKUP_RESULT`) and thus save some memory.
But adding a list is okay for me as well.

Reply:
Okay good point, let's keep the boolean flag then.